src/DataModelRegistry.hpp

```diff
@@ -26,9 +26,9 @@ class NODE_EDITOR_PUBLIC DataModelRegistry
 
   struct TypeConverterItem
   {
-    RegistryItemPtr Model;
-    NodeDataType    SourceType;
-    NodeDataType    DestinationType;
+    RegistryItemPtr Model{};
+    NodeDataType    SourceType{};
+    NodeDataType    DestinationType{};
```

Comment:
I think I mistakenly assumed that `std::unique_ptr` and a struct of two `std::string` are properly initialized without special statements, i.e. default-constructor for smart pointer would initialize it to `nullptr` and  strings will be empty.